EIPS/eip-4804.md

```diff
@@ -53,7 +53,7 @@ where
 
 ### Resolve Mode
 
-Once the "To" address and chainid are determined, the protocol will check the resolver mode of contract by calling "resolveMode" method. The protocol currently supports two resolve modes:
+Once the "To" address and chainid are determined, the protocol will check the resolver mode of contract by calling "resolveMode() returns (bytes32)" method. The protocol currently supports two resolve modes:
```

Comment:
Thanks for the comment.  So I think the resolve model should work like this:
- manual model if `resolveMode()` returns "manual"; or
- auto model if `resolveMode()` returns "auto" or "" or error; otherwise
- error with "unsupported resolve mode".

Note that if `resolveMode()` returns error, we still treat it as auto model since some existing contracts will throw if the method is not supported.